dashboards/data-ingest-entity-breakdown/data-ingest-entity-breakdown.json

```diff
@@ -1,282 +1,282 @@
+            "legend": {
+              "enabled": true
             },
-            "linkedEntityGuids": null
-          }
-        ]
-      }
-    ]
-  }
\ No newline at end of file
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM K8sClusterSample, K8sContainerSample,K8sDaemonsetSample, K8sDeploymentSample, K8sEndpointSample, K8sHpaSample, K8sNamespaceSample, K8sNodeSample, K8sPodSample, K8sReplicasetSample, K8sServiceSample, K8sVolumeSample  select bytecountestimate()/10e08 as 'GB Ingest' facet clusterName limit 100 TIMESERIES "
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        }
+      ]
+    },
+    {
+      "name": "Infrastructure Integrations",
+      "description": null,
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 9,
+            "width": 12
+          },
+          "title": "",
+          "rawConfiguration": {
+            "text": "## Install the Cloud Integration Dashboard\n\nCloud Integrations can often be the source of data ingest growth.  Without good visualizations it can be very difficult to pinpoint where the growth is coming from.  This is partly because these integrations are so easy to configure and they are not part of an organizations normal CI/CD pipeline and may also not be part of a formal configuration management system.\nFortunately this powerful set of dashboards can be installed [directly from New Relic I/O](https://onenr.io/0gR7VXxnnwo).\nIndividual dashboards installed by this pakage include:\n\n- AWS Integrations\n- Azure Integrations\n- GCP Integrations\n- On-Host Integrations\n- Kubernetes\n\n\n![AWS Integration Breakdown](https://raw.githubusercontent.com/newrelic/newrelic-quickstarts/v0.99.1/quickstarts/audit/infrastructure-integrations-data-analysis/dashboards/aws-integrations-data-ingest-analysis.png)"
```

Comment:
Updated this text here with a new link

Reply:
https://onenr.io/0gR7VXxnnwo Upgrading to this link to navigate to the new installation experience.